docs/dev/testing.rst

```diff
@@ -261,3 +262,45 @@ database containing all years of FERC 1 data (equivalent to ``tox -e ferc1_schem
 .. code-block:: console
 
   $ pytest --etl-settings test/settings/full-integration-test.yml test/integration/etl_test.py::test_ferc1_schema
+
+-------------------------------------------------------------------------------
+Data Validation
+-------------------------------------------------------------------------------
+Once the PUDL ETL pipeline has processed all of the available data, we have a
+collection of tests that can be run to verify a collection of expectations
```

Comment:
Would be good to explain whether/when they are run on our end. Are these tests part of the ETL? When do we run them to ensure the published output we provide is good to go? Should users be able to trust that they have already been run if they are using the docker? When should they consider running these tests for themselves?

Reply:
Added a note about this.